src/Shaders/default.fragment.fx

```diff
@@ -255,6 +255,9 @@ void main(void) {
 
 #ifdef LIGHTMAP
 	vec3 lightmapColor = texture2D(lightmapSampler, vLightmapUV + uvOffset).rgb * vLightmapInfos.y;

```

Comment:
The `* vLightmapInfos.y` multiplication should be done after the `#ifdef RGBDLIGHTMAP` block.

Reply:
good catch.. can you PR pretty please?

Reply:
@MackeyK24 beat me to it!